docs/guides/integrating-the-feature-hub.md

```diff
@@ -75,11 +75,13 @@ which are also build using React on the server as well as on the client, while
 it can still integrate Feature Apps that are built using other technologies on
 the client.
 
-| Integrator Package | Feature App Type  | Universal SSR | Built-In Shadow DOM | Required Integrator UI Library | Supported Feature App UI Libraries |
-| ------------------ | ----------------- | ------------- | ------------------- | ------------------------------ | ---------------------------------- |
-| @feature-hub/react | React Feature App | ✅            | ❌                  | `react@^16.3.0`                | `react@^16.3.0`                    |
-|                    | DOM Feature App   | ❌            | ❌                  | `react@^16.3.0`                | ✅ all                             |
-| @feature-hub/dom   | DOM Feature App   | ❌            | ✅                  | ✅ None                        | ✅ all                             |
+| Integrator Package                 | @feature-hub/react |                 | @feature-hub/dom |
+| ---------------------------------- | ------------------ | --------------- | ---------------- |
+| Feature App Type                   | `ReactFeatureApp`  | `DomFeatureApp` | `DomFeatureApp`  |
```

Comment:
Now that the “features” aren’t in the table header anymore but in the first column instead, do we want to make them bold manually?

Reply:
done